Osiris/Hacks/AntiAim.cpp

```diff
@@ -1,28 +1,69 @@
 #include "AntiAim.h"
+#include "../Memory.h"
 #include "../Interfaces.h"
 #include "../SDK/Engine.h"
 #include "../SDK/Entity.h"
 #include "../SDK/EntityList.h"
 #include "../SDK/NetworkChannel.h"
 #include "../SDK/UserCmd.h"
+#include "../SDK/GlobalVars.h"
 
-void AntiAim::run(UserCmd* cmd, const Vector& previousViewAngles, const Vector& currentViewAngles, bool& sendPacket) noexcept
+bool Lby()
+{
+	constexpr auto timeToTicks = [](float time) {  return static_cast<int>(0.5f + time / memory->globalVars->intervalPerTick); };
+	auto serverTime = memory->globalVars->serverTime();
+	static float nextLby, lastLby;
+
+	if (!(localPlayer->flags() & PlayerFlags::ONGROUND))
+	{
+		return false;
+	}
+	if (localPlayer->velocity().notNull() > 0.1f)
```

Comment:
Why are you comparing a bool to a float? Did you mean `localPlayer->velocity().length()`?